makefile

```diff
@@ -34,6 +34,7 @@ clean:
 
 lint:
 	$(JSHINT) src/gl/*.js
+	$(JSHINT) src/styles/*.js
```

Comment:
Back when I setting this up I was having issues with the recursive option, but it seems to work now. 

Will fix.